Solutions/CTERA/Data Connectors/CTERA_Data_Connector.json

```diff
@@ -0,0 +1,133 @@
+{
+    "id": "CTERA",
+    "title": "CTERA Syslog",
+    "publisher": "CTERA",
+    "description": "Ingest CTERA syslog data into Microsoft Sentinel",
+    "additionalRequirementBanner": "None ",
```

Comment:
Hello @roberteliass, Please add a short description markdown to the data connector. 
For ref: https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Theom/Data%20Connectors/Theom.json

Reply:
Updated, will commit shortly